Project.toml

```diff
@@ -1,18 +1,18 @@
 name = "NNlib"
 uuid = "872c559c-99b0-510c-b3b7-b6c96a88d5cd"
-version = "0.6.6"
+version = "0.6.7"
```

Comment:
I think a restriction of allowed Julia versions warrants a minor version bump

Reply:
I think it's fine, julia users on any version won't see any breaking change. Propagating to the ecosystem a minor version update is slightly annoying and I am always happy to avoid it

Reply:
I would suggest a version bump here too if it changes underlying behaviour of a package. In this case the pr makes use of nnpack by default, so should be reflected in versioning.

Reply:
> propagating to the ecosystem a minor version update is slightly annoying
That was my thought as well. 

>In this case the pr makes use of nnpack by default
I'm not sure I understand how that worked.  I thought the current code is functionally equivalent -- if the binaries are present, they'll be used. Just that binaries are now available for more platforms. But maybe I'm misunderstanding something.

Reply:
Quoting from https://github.com/JuliaRegistries/RegistryCI.jl/blob/729889777565fb0275110034ee933bd509b8962e/src/AutoMerge/guidelines.jl#L94-L100

> A patch release is not allowed to narrow the supported ranges of Julia versions

Reply:
> Propagating to the ecosystem a minor version update is slightly annoying and I am always happy to avoid it

This is the problem of being in 0.x for too long.  Narrowing the supported ranges of Julia versions always requires a minor version bump, but in 1+ series this is not a big deal

Reply:
> Quoting from https://github.com/JuliaRegistries/RegistryCI.jl/blob/729889777565fb0275110034ee933bd509b8962e/src/AutoMerge/guidelines.jl#L94-L100
> 
> > A patch release is not allowed to narrow the supported ranges of Julia versions

this applies only to `if (old_version >= v"1") || (new_version >= v"1")` though

Reply:
Uhm, right :sweat_smile: